ci-operator/jobs/open-cluster-management/multicloud-operators-foundation/open-cluster-management-multicloud-operators-foundation-release-2.1-postsubmits.yaml

```diff
@@ -0,0 +1,90 @@
+postsubmits:
+  open-cluster-management/multicloud-operators-foundation:
+  - agent: kubernetes
+    branches:
+    - ^release-2\.1$
+    cluster: api.ci
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/is-promotion: "true"
+      ci-operator.openshift.io/prowgen-controlled: "true"
+    max_concurrency: 1
+    name: branch-ci-open-cluster-management-multicloud-operators-foundation-release-2.1-images
+    spec:
+      containers:
+      - args:
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --oauth-token-path=/usr/local/github-credentials/oauth
+        - --promote
+        - --report-password-file=/etc/report/password.txt
+        - --report-username=ci
+        - --target=[images]
+        command:
+        - ci-operator
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /usr/local/github-credentials
+          name: github-credentials-openshift-ci-robot-private-git-cloner
+          readOnly: true
+        - mountPath: /etc/pull-secret
+          name: pull-secret
+          readOnly: true
+        - mountPath: /etc/report
+          name: result-aggregator
+          readOnly: true
+      serviceAccountName: ci-operator
+      volumes:
+      - name: github-credentials-openshift-ci-robot-private-git-cloner
+        secret:
+          secretName: github-credentials-openshift-ci-robot-private-git-cloner
+      - name: pull-secret
+        secret:
+          secretName: regcred
+      - name: result-aggregator
+        secret:
+          secretName: result-aggregator
+  - agent: kubernetes
+    branches:
+    - ^release-2\.1$
+    cluster: api.ci
+    decorate: true
+    name: branch-ci-open-cluster-management-multicloud-operators-foundation-release-2.1-publish
+    spec:
+      containers:
+      - args:
+        - -f
+        - /opt/build-harness/Makefile.build-harness-openshift-ci
+        - pipeline-manifest/update
+        command:
+        - make
+        env:
+        - name: COMPONENT_NAME
+          value: multicloud-operators-foundation
```

Comment:
are you sure it is the right component name? please check the pipeline repo at first. I think it should be mulitcloud-manager?